pvt/common/hardhat-base-config.ts

```diff
@@ -62,6 +62,7 @@ export const warnings = {
   // that.
   'contracts/test/**/*': {
     'code-size': 'off' as WarningRule,
+    'func-mutability': 'warn' as WarningRule,
```

Comment:
Unfortunately, not. The code can't compile because computeInvariantMock/computeBalanceMock needs to be `view` but the compiler argues that that needs to be pure.

Reply:
Could you make it read a dummy storage variable in the mock or something, so that it really is "view"? Like declare a numeric (that will initialize to zero), and add it to the weights?

Reply:
yeah I'd rather do something at the mock level even if it's not needed and remove this rule.